import org.apache.hadoop.io.retry.RetryPolicies.FailoverOnNetworkExceptionRetry;
public class FunctionUse4284 {
public void funcUse() {
FailoverOnNetworkExceptionRetry failoveronnetworkexceptionretry = new FailoverOnNetworkExceptionRetry();
}
}